.opencode/skills/react-native-best-practices/references/bundle-barrel-exports.md

```
@@ -0,0 +1,248 @@
+// - Button.tsx
+// - Card.tsx
+// - Modal.tsx  
+// - Sidebar.tsx
+// Even though you only use Button
+```
+
+### 3. Circular Dependencies
+
+Barrel files make cycles easier to create accidentally:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)